Firestore/src/Query.php

```diff
@@ -362,13 +362,17 @@ public function where($fieldPath, $operator, $value)
                 ]
             ];
         } else {
+            $encodedValue = ($operator == FieldFilterOperator::IN
+                ? $this->valueMapper->encodeMultiValue((array)$value)
+                : $this->valueMapper->encodeValue($value)
+            );
```

Comment:
Why is this wrapped in `()`? I don't see why it's necessary.

Reply:
Well, yes, it's not necessary, just a habit. Two reasons:
- I always wrap logical expressions (including ternary operators) to avoid surprises with precedence
- I just don't like it when indentation level changes suddenly

If those braces violate style guides, I'll remove them.

Reply:
Fair enough habit, but while it's not a hard style guide requirement, omitting braces where not required is standard throughout the library and I'd like to remain consistent.

```php
            $encodedValue = $operator == FieldFilterOperator::IN
                ? $this->valueMapper->encodeMultiValue((array)$value)
                : $this->valueMapper->encodeValue($value);
```